dhash/module.py

```diff
@@ -45,6 +64,42 @@ def _in_repost_channel(self, message: nextcord.Message) -> bool:
     async def dhash(self, ctx):
         await utils.discord.send_help(ctx)
 
+    @commands.guild_only()
+    @commands.check(check.acl)
+    @dhash.group(name="regex")
+    async def dhash_regex(self, ctx):
+        await utils.discord.send_help(ctx)
+
+    @commands.check(check.acl)
+    @dhash_regex.command(name="unset")
+    async def dhash_regex_unset(self, ctx):
+        HashConfig.set("allowed_urls", None)
+        self.allowed_urls = None
+        await ctx.reply(_(ctx, "Regex successfuly unset."))
```

Comment:
There must be guild info log about this.

Reply:
Gonna fix that.